docs/.vuepress/config.js

```diff
@@ -165,6 +165,23 @@ module.exports = {
             },
           ]
         },
+        {
+          title: "IBC Middleware Modules",
```

Comment:
In the codebase we has the fee middleware module inside the `apps` folder but here we would have the documentation under `middleware`. Are we ok with having this kind of distinction? Or should we move the module to a new `middleware` folder under `modules`?

Reply:
I don't feel particularly strongly about either option. It would be fine to leave under apps for me as its "application middleware" but I'm easy.